tensorboard/components/experimental/plugin_util/core-host-impl.ts

```diff
@@ -43,5 +50,18 @@ export class PluginCoreApiHostImpl {
       }
       return result;
     });
+
+    this.store
+      .select(getPluginsListLoaded)
```

Comment:
While `getPluginsListLoaded` changes happen to correspond exactly(?) to reload/manualReload actions today, these seem like fundamentally different actions.  e.g. in the future, the contract between updating the plugins list and reload events can change in another file, and the author may not know that this file also needs to change.

Could we add a `lastReloadTimeInMs` in the store?  With that, we could also guarantee that the callback will never fire on the initial load.

Reply:
> Could we add a lastReloadTimeInMs in the store? 

Sorry, I don't follow. It is already a state from the store?

Reply:
No, it does not exist currently.  I am suggesting that we should add this new field in the Core store's state along with the appropriate reducer and selector, so that core-host-impl can do `this.store.select(getLastReloadTimeInMs)`.

Less preferably, if we want plugin listing loads to be bound 1 to 1 to reloads, then we should explicitly add a comment in `webapp/core/actions/core_actions.ts` next to `pluginsListingRequested`, so that anyone who wants to use it understands that it must only ever be dispatched on coreLoaded, reload, manualReload.

Reply:
Ah, I wasn't sure what was being communicated here. You are protesting against using the `PluginListing` as a proxy to figuring out whether the application load state. Yes, that's correct and that's something to be fixed eventually.

To move forward, since that refactor actually has larger impact than just scope of this PR (plugins_container is actually using that proxy state incorrectly), I added a selector that is to be later correctly implemented. Does that sound good?

Reply:
Thanks, this sounds good to me.  From the CoreHostImpl's perspective, it is nicely unaware of the pluginsListing!